tests/lib/cli.js

```diff
@@ -1160,4 +1161,29 @@ describe("cli", () => {
         });
     });
 
+    describe("testing the cloned config", () => {
```

Comment:
Can we add a comment linking to the issue? I think that will provide enough context. :)

Reply:
sure 👍